packages/side-nav/test/side-nav-item.test.js

```diff
@@ -283,6 +284,47 @@ describe('side-nav-item', () => {
     });
   });
 
+  describe('matchExact', () => {
+    let currentPath = '/';
+    let pathnameStub;
+
+    beforeEach(() => {
+      pathnameStub = sinon.stub(location, 'pathname').get(() => currentPath);
+    });
+    afterEach(() => {
```

Comment:
nit: let's add a newline between `beforeEach` and `afterEach`.

Reply:
Done.